src/blob_format.cc

```diff
@@ -171,6 +172,26 @@ Status BlobFileMeta::DecodeFrom(Slice* src) {
   return Status::OK();
 }
 
+Status BlobFileMeta::DecodeFromV3(Slice* src) {
```

Comment:
Too many duplicated code with V1&V2.

Reply:
Better keep the original logic of V1 and V2 unchanged

Reply:
The main reason is I put the new `block_size` in between existing fields, instead of the end of all fields. So, it is a little bit tricky to reuse the code